### PR TITLE
fix: Update Colors In `CameraKeyboardViewController`, `CameraKeyboardPermissionsCell`

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we update the colors in `CameraKeyboardViewController` and `CameraKeyboardPermissionsCell` so we use SemanticColors instead of the deprecated ones.

In addition to that we add a bit of documentation in the classes we touched as well as updating the snapshot tests

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 1]
```diff
@@ -86,6 +86,7 @@ public enum SemanticColors {
         static let foregroundDefault = UIColor(light: Asset.Colors.gray90, dark: Asset.Colors.white)
         static let foregroundDefaultBlack = UIColor(light: Asset.Colors.black, dark: Asset.Colors.white)
         static let foregroundDefaultWhite = UIColor(light: Asset.Colors.white, dark: Asset.Colors.black)
+        static let foregroundWhite = UIColor(light: Asset.Colors.white, dark: Asset.Colors.white)
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift:89]: Same here, we have `foregroundDefaultWhite`

File: wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 2]
```diff
@@ -100,6 +101,8 @@ public enum SemanticColors {
 
     public enum View {
         static let backgroundDefault = UIColor(light: Asset.Colors.gray20, dark: Asset.Colors.gray100)
+        static let backgroundBlack = UIColor(light: Asset.Colors.black, dark: Asset.Colors.black)
+        static let backgroundtWhite = UIColor(light: Asset.Colors.white, dark: Asset.Colors.white)
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]: Is semantic color necessary? It's just white and black

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]: I was thinking about that. And I think I'm planning to remove those colors and use black and white instead.

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]: Done!

File: wire-ios/Wire-iOS Tests/TestPlans/AllTests.xctestplan [hunk 3]
```diff
@@ -40,6 +40,9 @@
   },
   "testTargets" : [
     {
+      "skippedTests" : [
+        "ShareViewControllerTests\/testThatItRendersCorrectlyShareViewController_Photos()"
```
Comment (netbe) [inline wire-ios/Wire-iOS Tests/TestPlans/AllTests.xctestplan]: did you add that to quarantine list?

**Review by netbe — APPROVED** [review]
LGTM;) just add the skipped tests to the quarantine list in our jira board

File: wire-ios/Wire-iOS Tests/TestPlans/AllTests.xctestplan (hunk 3 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS Tests/TestPlans/AllTests.xctestplan]: Yes!

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**